libr/core/cmd.c

```diff
@@ -1821,6 +1821,16 @@ static int cmd_resize(void *data, const char *input) {
 			}
 		}
 		return true;
+	case 'j': // "rj"
+		if (core->file) {
```

Comment:
Is this check really necessary?

Reply:
Yeah, I thought so too, but when I remove that, I get a:
```
error: a label can only be part of a statement and a declaration is not a statement
 1826 |   PJ * pj = pj_new ();
```

Reply:
When I simply put them, `rj` returned: 
`r_io_resize: cannot resize`

Reply:
what do you mean by "put them"?

Reply:
Sorry.
What I meant was, when I had used them without the check, inside a curly bracket, that particular error was displayed.

Now, I reckon that, this error is not related to checking for `core->file`.